Change Security Administrator -> Security Manager

tomkerkhove opened the pull request:
Change Security Administrator -> Security Manager since that's what Azure Security Center is using

azurecla commented:
Hi __@mariusrusu__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

davidwrede commented:
Hi @mariusrusu, thanks for the updates! I'll get this merged in and it should show up tomorrow morning.

Thanks,
Dave

tomkerkhove committed:
Change Security Administrator -> Security Manager

Change Security Administrator -> Security Manager since that's what Azure Security Center is using
